code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -178,3 +178,43 @@ var/global/list/blood_list = list()
 	random_icon_states = list("mucus")
 
 	var/dry=0
+
+/obj/effect/decal/cleanable/blood/gibs/blood
+	name = "blood"
+	icon = 'icons/effects/blood.dmi'
+	random_icon_states = list("mfloor1", "mfloor2", "mfloor3", "mfloor4", "mfloor5", "mfloor6", "mfloor7")
+	amount = 5
+	base_icon = 'icons/effects/blood.dmi'
+
+/obj/effect/decal/cleanable/blood/gibs/blood/update_icon()
```

Comment:
It is a copypaste. There are some relevant vars that I could've changed like `amount` but I want those ones this way.
So the idea was: I wanted the effect `proc/gibs` has, but with blood (no gibs). At the end, I found out that blood decals could NOT be moved, and I still don't know why, what's so different about them. To work around this, I created a child of the gibs decal, since they can be moved properly.
It now works and I added some notes about them so people can make blood messes more easily.

This was suggested by another coder.